components/automate-ui/src/app/pages/automate-settings/automate-settings.component.ts

```diff
@@ -178,6 +179,19 @@ export class AutomateSettingsComponent implements OnInit, OnDestroy {
     return allowedKeys.includes(key);
   }
 
+  // Update the width of input when greater than one number
```

Comment:
Much less confusing this way:
```suggestion
  // Update the width of input when greater than one digit
```